Makefrag

```diff
@@ -34,6 +34,8 @@ HARNESS_FILE ?=$(build_dir)/$(long_name).harness.v
 SMEMS_FILE   ?=$(build_dir)/$(long_name).mems.v
 SMEMS_CONF   ?=$(build_dir)/$(long_name).mems.conf
 sim_dotf ?= $(build_dir)/sim_files.f
+sim_harness_blackboxes ?= $(build_dir)/firrtl_black_box_resource_files.harness.f
+sim_top_blackboxes ?= $(build_dir)/firrtl_black_box_resource_files.top.f
```

Comment:
Is this always put here, or do we pass this path in as an option? (I don't see the latter anywhere)

Reply:
I mirrored the other .f file above which is always put in the same place.
Unfortunately firrtl is dumb here https://github.com/freechipsproject/firrtl/blob/97e132e24226e69afc3a52d88c4a591b0678f816/src/main/scala/firrtl/transforms/BlackBoxSourceHelper.scala#L154
And always puts the .f file in the same name so we are doing the management ourselves. You could change this variable if you'd like them to live somewhere else.

Reply:
ok, makes sense.